### PR TITLE
[CI] Add yaml linter

### PR DESCRIPTION
**Pull request by Tankilevitch**
# Description

What - Add yaml linter
Why - Releasing azure integration failed due to yaml formatting that wasn't detected
How - add yamllint package as part of our makefile

## Type of change

Please leave one option from the following and delete the rest:

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: integrations/azure/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+0.1.1 (2023-08-20)
```
Comment (yairsimantov20) [inline integrations/azure/CHANGELOG.md:10]: did u generate this using towncrier?
it looks like it missing the `#`

File: integrations/azure/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Bug Fixes
+
+- Remove capability to remove port entity on received event of resource deletion (#1)
```
Comment (yairsimantov20) [inline integrations/azure/CHANGELOG.md:14]:
```suggestion
- Remove capability to remove port entity on received event of resource deletion
```

File: integrations/azure/poetry.lock
Comment (yairsimantov20) [inline integrations/azure/poetry.lock:1]: Why is there a change in this file?

**Review by yairsimantov20 — APPROVED** [review]

File: changelog/1.improvement.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Add yaml linter
```
Comment (MPTG94) [inline changelog/1.improvement.md:1]:
```suggestion
Added yaml linter
```

File: integrations/azure/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# 0.1.1 (2023-08-20)
+
+### Bug Fixes
+
+- Remove capability to remove port entity on received event of resource deletion
```
Comment (MPTG94) [inline integrations/azure/CHANGELOG.md:14]:
```suggestion
- Removed capability to remove port entity on received event of resource deletion
```

**Review by MPTG94 — APPROVED** [review]
Reviewed the .md files

**Pull request merged**